src/components/ChatEntry.js

```diff
@@ -1,22 +1,38 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+
+
+const ChatEntry = ({id, sender, body, timeStamp, liked, heartClick}) => {
+
+  const clickLikedHeart = () => {
+    heartClick(
+      {id:id,
+       sender:sender,
+       body:body, 
+       timeStamp:timeStamp, 
+       liked:!liked})};
 
-const ChatEntry = (props) => {
   return (
     <div className="chat-entry local">
```

Comment:
While it is an optional enhancement to make the[ local and remote messages](https://github.com/Ada-C19/react-chatlog/blob/main/project-docs/optional-enhancements.md#local-and-remote-messages) display on different sides of the screen, I did want to plant the seed of how you could do that. 

If you look in ChatEntry.css, there are 2 classes called 'local' and 'remote'. Right now, on line 18 we've hardcoded one of the class names to be 'local'. But what if we checked to see if the `sender` === 'Vladimir' and assign a variable to be 'remote' and if it wasn't Vladimir, then the variable would be 'local'. 

`const senderClass = sender === 'Vladimir' ? 'remote' : 'local';`

Then you could use senderClass on line 18 to set the class depending on the sender, like:
```
<div className={`chat-entry ${senderClass}`}>